docs/03-tutorials/00-application-connectivity/ac-05-call-registered-service-from-kyma.md

```diff
@@ -53,7 +51,6 @@ This guide shows how to call a registered external service from Kyma using a sim
    kind: Function
    metadata:
      name: my-function
-     namespace: $NAMESPACE
```

Comment:
I was under the impression that the Namespace you provided here was connected with the Namespace in which you created your Application, and since you don't need to bind your Application with any Namespace anymore, you no longer need it. I see my mistake now. I'll bring the **namespace** field back.